components/script/dom/document.rs

```diff
@@ -3848,12 +3838,13 @@ impl Document {
     }
 
     pub fn set_referrer_policy(&self, policy: Option<ReferrerPolicy>) {
-        self.referrer_policy.set(policy);
+        self.policy_container
+            .borrow_mut()
+            .set_referrer_policy(policy);
     }
 
-    //TODO - default still at no-referrer
     pub fn get_referrer_policy(&self) -> Option<ReferrerPolicy> {
```

Comment:
If we always return Some here, let's remove the Option from the return type.

Reply:
👍 updated in https://github.com/servo/servo/pull/34263/commits/3104746061f18756021e1ab177820e3d9c8f38fc

[Updated try](https://github.com/shanehandley/servo/actions/runs/11886854303)